test-integration/graql/query/BUILD

```diff
@@ -20,161 +20,159 @@ load("@graknlabs_build_tools//checkstyle:rules.bzl", "checkstyle_test")
 
 java_test(
     name = "graql-get-it",
-    test_class = "grakn.core.graql.query.GraqlGetIT",
+    size = "medium",
     srcs = ["GraqlGetIT.java"],
+    test_class = "grakn.core.graql.query.GraqlGetIT",
     deps = [
-        "//common:common",
-        "//concept:concept",
-        "//server:server",
+        "//common",
+        "//concept",
+        "//server",
         "//test-integration/graql/graph:movie-graph",
         "//test-integration/rule:grakn-test-server",
-
         "@graknlabs_graql//java:graql",
     ],
-    size = "medium"
 )
 
 java_test(
     name = "graql-define-it",
-    test_class = "grakn.core.graql.query.GraqlDefineIT",
+    size = "medium",
     srcs = ["GraqlDefineIT.java"],
+    test_class = "grakn.core.graql.query.GraqlDefineIT",
     deps = [
-        "//common:common",
-        "//concept:concept",
-        "//server:server",
+        "//common",
+        "//concept",
+        "//dependencies/maven/artifacts/org/hamcrest:hamcrest-library",
+        "//server",
         "//test-integration/graql/graph:movie-graph",
         "//test-integration/rule:grakn-test-server",
         "//test-integration/util:graql-test-util",
-
         "@graknlabs_graql//java:graql",
-
-        "//dependencies/maven/artifacts/org/hamcrest:hamcrest-library",
     ],
-    size = "medium"
 )
 
 java_test(
     name = "graql-delete-it",
-    test_class = "grakn.core.graql.query.GraqlDeleteIT",
+    size = "medium",
     srcs = ["GraqlDeleteIT.java"],
+    test_class = "grakn.core.graql.query.GraqlDeleteIT",
     deps = [
-        "//common:common",
```

Comment:
you should put back the target names, @kasper-piskorski . As you can see in other target definitions, we're making it a convention to write the full `//package/name:target-name`

Reply:
it's not me, it's bazel/intellij